Tribler/Test/Core/Modules/MetadataStore/test_channel_metadata.py

```diff
@@ -374,3 +442,55 @@ def test_add_torrents_from_dir(self):
 
     def test_add_torrents_from_dir_recursive(self):
         self.check_add(11, 1, recursive=True)
+
+    @db_session
+    def test_make_copy(self):
```

Comment:
I always add a docstring above every test to quickly explain what it is testing (and sometimes what I expect). This docstring is displayed when running the tests. This might be a matter of personal preference but I recommend annotating these tests with such comments. The reason is that the (limited length of the) method name might not always reflect the right intentions of the test.

Reply:
Ok. I'll add the strings to the tests with short names